ocs_ci/ocs/amq.py

```diff
@@ -759,3 +760,36 @@ def cleanup(
         # Reset namespace to default
         switch_to_default_rook_cluster_project()
         self.ns_obj.wait_for_delete(resource_name=kafka_namespace)
+
+
+def get_node_objs_where_benchmark_pod_not_hosted(
+    benchmark_pod_name="benchmark", namespace="tiller"
+):
+    """
+    Return node list obj where benchmark pod not hosted
+
+    Args:
+        benchmark_pod_name (str): Name of the benchmark pod
+        namespace (str): Namespace of benchmark pod hosted
+
+    Returns:
+        list: Cluster node OCP objects
+
+    """
+    # Get the worker node list
+    worker_nodes = get_typed_nodes(node_type='worker')
+
+    # Get the node name
+    worker_node = [node.get().get('metadata').get('name') for node in worker_nodes]
```

Comment:
the var should reflect that the value is a list.
As you are not using `worker_nodes` var in other places, I think you can do:
```python
worker_nodes = [node.get().get('metadata').get('name') for node in get_typed_nodes(node_type='worker')]
```